.github/workflows/ft_tests.yml

```diff
@@ -0,0 +1,40 @@
+name: Free Threading Tests
+on: [push, pull_request]
+jobs:
+  tests:
```

Comment:
Nit: could you change this to `tests_freethreading` or another more descriptive name? When you look at the CI log for the run, it's otherwise not visible that that is a free-threading run.

Reply:
Yup, done, `freethreading_tests` (alphabetical..)